genotype/store/cli.py

```diff
@@ -105,6 +109,28 @@ def sample(context, sample_id):
         context.abort()
 
 
+@click.command('prepare-trending')
+@click.option('-s', '--sample-id', 
+                help = 'return sample with specific sample id.')
+@click.option('-d', '--days',
+                help = 'return samples added a specific number of days ago.')
+@click.pass_context
+def prepare_trending(context, days, sample_id):
```

Comment:
Why do we have a prepare-method?

Reply:
To restructure the sql data into a mongo object witch can be passed into vogue. Is that wrong??

Reply:
i.e genotype would benefit if we move it from vogue from genotype